yoga/Yoga.cpp

```diff
@@ -1007,7 +1007,7 @@ bool YGNodeLayoutGetDidLegacyStretchFlagAffectLayout(const YGNodeRef node) {
   return node->getLayout().doesLegacyStretchFlagAffectsLayout;
 }
 
-uint32_t gCurrentGenerationCount = 0;
+std::atomic<uint8_t> gCurrentGenerationCount(0);
```

Comment:
Again, this causes additional synchronisation, and I am not sure it’s necessary:

this will usually be loaded into a register, incremented, and written back to memory.

When two threads do this in parallel, they might both load the old value, increment by one, and write back – with the global value being incremented by one, not two, as a result.

That will still increment the count, and cause layouts to be run – there isn’t really a need to synchronise it.

Am I missing something?

Minor nitpick, please use uniform initialization (`std::atomic<uint8_t> gCurrentGenerationCount{0};`)